cmd/influx/pkg.go

```diff
@@ -42,9 +42,8 @@ type cmdPkgBuilder struct {
 	hasColor        bool
 	hasTableBorders bool
 	meta            pkger.Metadata
-	orgID           string
-	org             string
-	quiet           bool
+	organization
```

Comment:
lets not embed this here, namespace it by a field name, something as simple as the following would be fine.

```go
org organization
```

Reply:
fair enough!